demisto_sdk/tests/test_files/README-invalid.md

```diff
@@ -1,7 +1,10 @@
 Use the Quin-C AccessData integration to protect against and provide additional visibility into phishing and other malicious email attacks. This integration was integrated and tested with version 20190926 of Quin-C Accessdata.
 
 Documentation for the integration was provided by Quin-C.
-
+For more info check out think links-
```

Comment:
please also add tests for html. I think its no issue to be in the same file

Reply:
I would add a couple more html tests, like you have for markdown

Reply:
Added a couple more, the html files are old and I don't think we create more so I think it should be enough.